src/components/DelegationDashboard/DelegationWalletConnectModal.tsx

```diff
@@ -139,23 +150,56 @@ const WalletConnectModal = ({ className, open, setOpen, closable }: Props) => {
 
 	useEffect(() => {
 		getWallet();
-		loginWallet!==null && getAccounts(loginWallet);
+		if(loginWallet !==null){
+			getAccounts(loginWallet);
+			setWallet(loginWallet);
+		}
 	// eslint-disable-next-line react-hooks/exhaustive-deps
 	},[]);
 
+	const handleInitiatorBalance = useCallback(
+		async () => {
+			if (!api || !apiReady) {
+				return;
+			}
+			//deposit balance
+			const depositBase = api.consts.multisig.depositBase.toString();
```

Comment:
please check if these methods : `depositBase` and `depositFactor` etc are available for all polkasafe chains.

Reply:
null checks and other handling might be required